src/modules/messages/proxyPluralMessage.ts

```diff
@@ -37,7 +41,30 @@ export const proxyPluralMessage = async (
       allowedMentions: {
         parse: [],
       },
-    });
+      embeds: [],
+    };
+
+    if (message.type === MessageType.Reply && message?.reference?.messageId) {
+      const originalMsg = await message.channel.messages.fetch(
+        message.reference.messageId
+      );
+
+      content.content =
+        (message.mentions.users.size ? `<@${originalMsg.author.id}>, ` : "") +
+        `${content.content}`;
+      content.allowedMentions = {
+        users: [originalMsg.author.id],
+      };
+
+      content.embeds?.push(
+        new EmbedBuilder().setDescription(originalMsg.content).setAuthor({
+          name: originalMsg.author.username,
+          iconURL: originalMsg.author.displayAvatarURL(),
+          url: originalMsg.url,
+        })
+      );
+    }
```

Comment:
```suggestion
      content.embeds?.push(
        new EmbedBuilder()
          .setDescription(originalMsg.content)
          .setAuthor({
            name: originalMsg.author.username,
            iconURL: originalMsg.author.displayAvatarURL(),
            url: originalMsg.url,
          })
          .setURL(originalMsg.url)
      );
```

Last nitpick and should be good to go.